src/hooks/useDailyStatistics/utils.ts

```diff
@@ -1,16 +1,25 @@
 import { chain, sumBy } from "lodash";
 import { DailyStatistics, CampaignPolicy } from "../../types";
-import { formatQuantityText } from "../../components/CustomerQuota/utils";
 import { Sentry } from "../../utils/errorTracking";
 
+export type UnitType =
+  | {
+      type: "POSTFIX" | "PREFIX";
+      label: string;
+    }
+  | undefined;
+
+export type Transaction = {
+  name: string;
+  category: string;
+  quantity: number;
+  unitType: UnitType;
+  descriptionAlert?: "viaAppeal" | undefined;
+  order: number;
+};
```

Comment:
Thanks for the effort to refactor, but this is actually not a transaction. It's a combination of policies and the past transaction history.

Naming this transaction would conflict with the rest of the Transaction type in the repo as well.

If you would like to try to refactor, maybe we can do it in another PR? For now, I think it would be good to remove it. (: